src/components/RecentWorks.js

```diff
@@ -0,0 +1,269 @@
+    display: grid;
+    grid-template-column: 1fr;
+  }
+`;
+
+const Divider = styled.div`
+  position: absolute;
+  width: 894px;
+  height: 0px;
+  left: 380px;
+  top: 952px;
+
+  border: 1px solid #212121;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 1;
+  align-self: center;
+  flex-grow: 0;
+  margin: 24px 0px;
+
+  @media only screen and (max-width: 600px) {
+    width: 90%;
+    left: 10%;
+    top: 680px;
+    margin-left: -16px;
+  }
+
+  @media (min-width: 768px) and (max-width: 1024px) {
+    margin-left: -20px;
+  }
+`;
+
+const H1 = styled.h1`
+  position: static;
+  left: 0px;
+  top: 0px;
+
+  font-family: Crete Round;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 40px;
+  line-height: 52px;
+  /* identical to box height, or 130% */
+
+  display: flex;
+  align-items: center;
+
+  /* Neutrals/N900 */
+
+  color: #091E42;
+
+  /* Inside Auto Layout */
+
+  flex: none;
+  order: 0;
+  align-self: center;
+  flex-grow: 0;
+  margin: 24px 0px;
+
+  @media only screen and (max-width: 600px) {
+    position: relative;
+    width: 100%;
+    left: -40px;
+    top: 0px;
+
+    font-family: Crete Round;
+    font-style: normal;
+    font-weight: normal;
+    font-size: 34px;
+    line-height: 52px;
+    /* or 130% */
+
+    display: flex;
+    align-items: center;
+
+    /* Theme 3 / Neutral 1 */
+
+    color: #1C1A19;
+
+    /* Inside Auto Layout */
+
+    flex: none;
+    order: 0;
+    flex-grow: 0;
+    margin: 0px 24px;
+  }
+`;
+
+const SeePro = styled.button`
+  background: #545454;
+`;
+const ProjectW = styled.div`
+  width: 90%;
+  margin: auto;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+  }
+`;
+
+const Proj = styled.div`
+  @media only screen and (max-width: 600px) {
+   
+  }
+  
+`;
+const MobileImg = styled.div`
+  @media only screen and (max-width: 600px) {
+    position: relative;
+    left: 0px;
+    width: 100%;
+  }
+`;
+
+const Lang = styled.div`
+  height: 50px;
+  margin-top: 10px;
+  @media only screen and (max-width: 600px) {
+    width: 100%;
+    text-overflow: ellipsis;
+  }
+`;
+
+const TechStyle = styled.span`
+  border: 1px solid grey;
+  height: auto;
+`;
+const projects = [
+  {
+    image: '/Images/fb.png',
+    alt: 'Facebook App',
+    title: 'Facebook Clone',
+    description: 'An authenticated user can post, comment, send/accept/reject a friend request.',
+    codeLink: 'https://github.com/addod19/facebook-clone',
+    liveUrl: 'https://secret-sea-76381.herokuapp.com/',
+    tech: ['Rails', 'SCSS', 'OmniAuth', 'Bootstrap'],
+  },
+  {
+    image: '/Images/rec.png',
+    alt: 'Recipe App',
+    title: 'Recipe App',
+    description: 'This application allows any user to check for their favorite recipe and how to prepare.',
+    codeLink: 'https://github.com/addod19/redux_recipe',
+    liveUrl: 'https://redux-recipe.herokuapp.com/',
+    tech: ['React', 'CSS3', 'Jest', 'Axios'],
+  },
+  {
+    image: '/Images/ga.png',
+    alt: 'Shooter Game',
```

Comment:
[Optional]
- [ ] I think it would be better to have a separate file for the projects so we keep the codebase clean and simple :slightly_smiling_face:

Reply:
Very thoughtful optional change :+1: